tools/downloader/src/open_model_zoo/model_tools/download_engine/file_source.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import requests
+
+from open_model_zoo.model_tools.download_engine import validation
+
+DOWNLOAD_TIMEOUT = 5 * 60
```

Comment:
I think this is something user shall be able to control.

Reply:
added as a argument to `Downloader`